packages/pg/lib/client.js

```diff
@@ -243,6 +248,7 @@ class Client extends EventEmitter {
         }
       }
     }
+    logger.warn("Given key " + key + "is invalid")
```

Comment:
```suggestion
    logger.warn("Given topology-key " + key + " is invalid")
```

Reply:
I think in other drivers we fail in case of invalid values of these properties. But we can take it up separately.

Reply:
That happens for this driver also.